HACKING.md

```diff
@@ -242,6 +242,8 @@ If you want to store builds persistently, now is a good time to allocate
 S3 storage.  See the [upstream coreos-assembler docs](https://github.com/coreos/coreos-assembler/blob/main/README-design.md)
 around build architecture.
 
+If you need to set up versioning and lifecycle management for the bucket, refer to [s3 bucket versioning doc](https://github.com/coreos/fedora-coreos-pipeline/blob/main/docs/s3-bucket-versioning.md) for detailed steps. For FCOS builds, we’ll apply a lifecycle policy of 14 days to manage old versions efficiently.
+
```

Comment:
should we update the `aws s3 mb` example below to use the new API? `aws s3api` ?

Reply:
Obviously, the existing command should work but I am happy to update to new API in another commit.

Reply:
nvm - don't worry about this

Reply:
Hah, added the changes. Lmk they seem worth the change :eyes: